lib/lightning/pipeline.ex

```diff
@@ -11,6 +16,15 @@ defmodule Lightning.Pipeline do
   alias Lightning.Repo
   import Ecto.Query, only: [select: 3]
 
+  @impl Oban.Worker
+  def perform(%Oban.Job{args: %{"event_id" => id, "run_id" => _id}}) do
```

Comment:
I think we can just drop the `run_id` from this function, only really need the event.